monitor/dbt_project/models/alerts.sql

```diff
@@ -31,7 +38,12 @@ from all_alerts
 {%- if is_incremental() %}
 {%- set row_count = elementary.get_row_count(this) %}
     {%- if row_count > 0 %}
-        {# We use a backfill of 2 days to prevent race condition when multiple dbt tests + edr monitor run together #}
-        where detected_at > (select {{ elementary.timeadd('day', '-2', 'max(detected_at)') }} from {{ this }})
+        where detected_at > {{ last_alert_time_with_backfill }}
```

Comment:
Can't you remove the first `where` clause? You're double checking against `last_alert_time_with_backfill`.

Reply:
Nevermind 🙃